src/movies.js

```diff
@@ -1,16 +1,108 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
+function getAllDirectors(movies) {
+    let answer = movies.map(function(movie) {
```

Comment:
Try to give your variables more meaningful names of what is the value. For this case, something like directorsArr could be.